app/src/main/java/fr/gaulupeau/apps/Poche/data/Settings.java

```diff
@@ -351,7 +351,7 @@ public void setArticleFontSerif(boolean value) {
     }
 
     public int getArticlesListLimit() {
-        return getInt(R.string.pref_key_ui_lists_limit, 50);
+        return getInt(R.string.pref_key_ui_lists_limit, 100);
```

Comment:
This is unlikely to affect anything, not sure why I even bothered to set it. The actual default value is here: The actual default value is https://github.com/wallabag/android-app/blob/master/app/src/main/res/xml/preferences.xml#L96.